### ISSUE TITLE
Fails to build on MacOS

### ISSUE THREAD

**Issue by peregrinrm** [JuliaLang/julia#28632]
Hi!

I want to build the docs to have a local version, however building the docs seem to require building the whole system.

I have a Macbook Pro with MacOS High Sierra, with Xcode, and the julia build always fails with:

```
/Applications/Xcode.app/Contents/Developer/usr/bin/make -C scratch/openblas-e8a68ef261a33568b0f0cf53e0e2287e9f12e69e/ CC=clang -stdlib=libc++ -mmacosx-version-min=10.8 -m64 FC=gfortran -mmacosx-version-min=10.8 -m64 LD=ld RANLIB=ranlib TARGET= BINARY=64 USE_THREAD=1 GEMM_MULTITHREADING_THRESHOLD=50 NUM_THREADS=16 NO_AFFINITY=1 DYNAMIC_ARCH=1 INTERFACE64=1 SYMBOLSUFFIX=64_ LIBPREFIX=libopenblas64_ OBJCONV=/Users/peregrin/src/julia/deps/scratch/objconv/objconv CFLAGS= -O2 FFLAGS= -O2 -fPIC LDFLAGS= -Wl,-rpath,'@loader_path/' MAKE_NB_JOBS=0
Undefined symbols for architecture x86_64:
  "__gfortran_set_std", referenced from:
      _MAIN__ in sblat1.o
  "_main", referenced from:
     implicit entry/start for main executable
ld: symbol(s) not found for architecture x86_64
collect2: error: ld returned 1 exit status
make[3]: *** [sblat1] Error 1
make[2]: *** [tests] Error 2
*** Clean the OpenBLAS build with 'make -C deps clean-openblas'. Rebuild with 'make OPENBLAS_USE_THREAD=0' if OpenBLAS had trouble linking libpthread.so, and with 'make OPENBLAS_TARGET_ARCH=NEHALEM' if there were errors building SandyBridge support. Both these options can also be used simultaneously. ***
make[1]: *** [scratch/openblas-e8a68ef261a33568b0f0cf53e0e2287e9f12e69e/build-compiled] Error 1
make: *** [julia-deps] Error 2
```
--

My config:

```
cc --version
Apple LLVM version 9.1.0 (clang-902.0.39.2)
Target: x86_64-apple-darwin17.7.0
Thread model: posix

gfortran --version
GNU Fortran (GCC) 6.3.0
Copyright (C) 2016 Free Software Foundation, Inc.
This is free software; see the source for copying conditions.  There is NO
warranty; not even for MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.
```

Can you help me? 

TIA

Rodrigo

**Comment by ararslan** [comment]
I've submitted a PR (#29845) that updates the version of OpenBLAS used by Julia, which fixed this issue for me. Could you try building the `aa/openblas-0.3.3` branch to confirm that it fixes the issue for you as well?

**Comment by ararslan** [comment]
Should be fixed by #29845, which has now been merged. Let me know if you still can't build master.
